### ISSUE TITLE
sync/atomic: LoadPointer is missing for linux-amd64-race

### ISSUE THREAD

**Issue by bradfitz** [golang/go#9516]
Commit 7b4df8f018ec01df3ecbd9961c769e7199853363 broke the `linux-amd64-race` builder:

```
sync/atomic.(*Value).Load: call to external function sync/atomic.LoadPointer
sync/atomic.(*Value).Load: sync/atomic.LoadPointer: not defined
sync/atomic.(*Value).Load: sync/atomic.LoadPointer: not defined
sync/atomic.(*Value).Store: sync/atomic.LoadPointer: not defined
sync/atomic_test.TestLoadPointer: sync/atomic.LoadPointer: not defined
sync/atomic_test.hammerStoreLoadPointer: sync/atomic.LoadPointer: not defined
sync/atomic_test.func·038: sync/atomic.LoadPointer: not defined
sync/atomic.(*Value).Load: undefined: sync/atomic.LoadPointer
sync/atomic.(*Value).Load: undefined: sync/atomic.LoadPointer
sync/atomic.(*Value).Store: undefined: sync/atomic.LoadPointer
sync/atomic_test.TestLoadPointer: undefined: sync/atomic.LoadPointer
sync/atomic_test.hammerStoreLoadPointer: undefined: sync/atomic.LoadPointer
sync/atomic_test.func·038: undefined: sync/atomic.LoadPointer
```

/cc @dvyukov @rsc @minux

**Comment by cookieo9** [comment]
Windows/amd64 is also broken.

**Comment by minux** [comment]
Sent https://go-review.googlesource.com/2401.

./race.bash still has two failing tests after fixing the build,
but I think that's a separate issue.

**Comment by bradfitz** [comment]
Fixed by 5f179c7cef24c
